packages/payment-detection/test/any/any-to-erc20-proxy-info-retriever.test.ts

```diff
@@ -0,0 +1,124 @@
+// tslint:disable: no-invalid-this
+// tslint:disable: no-magic-numbers
+import { PaymentTypes, RequestLogicTypes } from '@requestnetwork/types';
+import AnyToErc20ProxyInfoRetriever from '../../src/any/any-to-erc20-proxy-info-retriever';
+import { ethers } from 'ethers';
+
+const erc20LocalhostContractAddress = '0x38cF23C52Bb4B13F051Aec09580a2dE845a7FA35';
+const conversionProxyContractAddress = '0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4';
+const erc20FeeProxyContractAddress = '0x75c35C980C0d37ef46DF04d31A140b65503c0eEd';
+const paymentReferenceMock = '0111111111111111111111111111111111111111111111111';
+const acceptedTokens = [erc20LocalhostContractAddress];
+
+/* tslint:disable:no-unused-expression */
+describe('api/any/conversion-proxy-info-retriever', () => {
```

Comment:
I don't understand this test, is it related to this payment network? 
`it('can get the localhost balance of an address'`
makes it look related to ERC20, for example.

Reply:
Can we discuss it IRL? I don't get it.